ztp/source-crs/AmqInstance.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: amq-router
   namespace: amq-router
   annotations:
-    ran.openshift.io/ztp-deploy-wave: "100"
+    ran.openshift.io/ztp-deploy-wave: "10"
```

Comment:
why this was updated

Reply:
amqInstance is in group PGT which has a wave of 10.

Reply:
removed wave change changes in AmqInstance.yaml and added annotations to update the wave in group PGT instead.
```
    # - fileName: AmqInstance.yaml
    #   policyName: "config-events-policy"
    #   annotations:
    #     ran.openshift.io/ztp-deploy-wave: "10"

```